go.mod

```diff
@@ -37,10 +37,10 @@ require (
 	github.com/imdario/mergo v0.3.16
 	github.com/klauspost/compress v1.17.2
 	github.com/klauspost/pgzip v1.2.6
-	github.com/mandelsoft/filepath v0.0.0-20230412200429-36b1eb66bd27
+	github.com/mandelsoft/filepath v0.0.0-20240223090642-3e2777258aa3
```

Comment:
@mandelsoft are you going to spend a new release for this?

Reply:
there will be a release for vfs